### DISCUSSION TITLE
SupabaseSessionHandler SessionRetriever not working in the example

### DISCUSSION THREAD

**Discussion by rhuanbarros** [supabase-community/supabase-csharp#42] [Q&A] [answered]
CustomSupabaseSessionHandler not working.
https://github.com/supabase-community/supabase-csharp/pull/40

- how to test: 

1. Navigate to: https://blazorwasmsupabasetemplate.web.app/crud-private
2. Logout
3. Login
4. Access Crud table private RLS page
5. Insert some row
6. you will see the row displayed below
7. press F5 on the browser
8. the rows will disapear

Maybe i created CustomSupabaseSessionHandler wrong. But it seem like that the token is not being sent with the headers in the request to get data from the table.

**Comment by rhuanbarros** [comment]
the header before press F5:

`authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJhdXRoZW50aWNhdGVkIiwiZXhwIjoxNjY5MTQzMDE0LCJzdWIiOiJmMDVmYWQ4Mi0xZDQwLTRhNzMtYWUxOS1hMzI1YzczMDQ4ZjgiLCJlbWFpbCI6ImNsaWVudGUxQGdtYWlsLmNvbSIsInBob25lIjoiIiwiYXBwX21ldGFkYXRhIjp7InByb3ZpZGVyIjoiZW1haWwiLCJwcm92aWRlcnMiOlsiZW1haWwiXX0sInVzZXJfbWV0YWRhdGEiOnt9LCJyb2xlIjoiYXV0aGVudGljYXRlZCIsInNlc3Npb25faWQiOiI0MzIyYmViOS1kY2M1LTQ2MTgtYmQyNi03YWQxYTI1OTdkNDgifQ.N5zPHziqKQKDU1NJnet-bl4MhSl8XeeKrZuOU6n1DRI
`

the header after press F5:

`authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InB5bG5lc2ZnbXl0amVnenpjdWxuIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NjgyOTMwMzcsImV4cCI6MTk4Mzg2OTAzN30.kI29Q_qYWDH5SD6oi5NTwHG6Pxy1e1AUfR8s_ga45lE
`

**Comment by acupofjose** [comment, marked as answer]
This was addressed in [ec5352065](https://github.com/supabase-community/supabase-csharp/pull/40/commits/ec535206509d3030a34d07882994a699d60ede41) of PR#40!
